python/tests/engine/test_python.py

```diff
@@ -1258,39 +1236,30 @@ def test_convert_simple_pandas_type_other(self):
 
         # Act
         with pytest.raises(ValueError) as e_info:
-            python_engine._convert_simple_pandas_dtype_to_offline_type(dtype="other")
+            python_engine._convert_simple_pandas_dtype_to_offline_type(
+                arrow_type="other"
+            )
 
         # Assert
         assert str(e_info.value) == "dtype 'other' not supported"
 
     def test_infer_type_pyarrow_list(self, mocker):
         # Arrange
-        mock_python_engine_convert_pandas_type = mocker.patch(
-            "hsfs.engine.python.Engine._convert_pandas_dtype_to_offline_type"
-        )
 
         python_engine = python.Engine()
 
-        mock_python_engine_convert_pandas_type.return_value = "test"
-
         # Act
         result = python_engine._convert_pandas_object_type_to_offline_type(
             arrow_type=pa.list_(pa.int8())
         )
 
         # Assert
-        assert result == "array<test>"
+        assert result == "array<int>"
 
     def test_infer_type_pyarrow_struct(self, mocker):
```

Comment:
one minor improvement for this test:
```
    def test_infer_type_pyarrow_struct(self):
        # Arrange
        python_engine = python.Engine()

        # Act
        result = python_engine._convert_pandas_object_type_to_offline_type(
            arrow_type=pa.struct([pa.field('f1', pa.int32())])
        )

        # Assert
        assert result == "struct<f1:int>"
```

Reply:
also optionally you can remove `mocker` args from test methods where they are not needed. (but ok if you wish to not do it :) )